xendit/tmpl/cc_payment.php

```diff
@@ -73,78 +79,86 @@
 
 		// Card validation on submit
 		var flag = false;
-
+		
+		$('#checkoutFormSubmit').attr('onclick','xenditClickCC()');
+		
 		$('#checkoutForm').submit(function(event) {
-			if (!flag) {
-				event.preventDefault();
-				var xendit_form = $('#checkoutForm');
-
-				var paymentType = $("input[name=virtuemart_paymentmethod_id]:checked").attr('xendit-payment-type');
-				if (paymentType == 'CC') {
-					var cardNumber = $('#xendit_gateway_card_number').val().replace(/\s/g, '');
-					var cardExpiry = $('#xendit_gateway_card_expiry').payment('cardExpiryVal');
-					var cardCode = $('#xendit_gateway_card_code').val();
-
-					var data = {
-						"card_number"   	: cardNumber,
-						"card_exp_month"	: String(cardExpiry.month).length === 1 ? '0' + String(cardExpiry.month) : String(cardExpiry.month),
-						"card_exp_year" 	: String(cardExpiry.year),
-						"card_cvn"      	: cardCode,
-						"is_multiple_use"	: true
-					};
-
-					Xendit.setPublishableKey('<?php echo $viewData['public_key']; ?>');
-
-					$('#year').val(data.card_exp_year);
-					$('#month').val(data.card_exp_month);
-					$('#card_cvn').val(data.card_cvn);
+			if (useClick){
+				useClick = false;
```

Comment:
@candrasaputra @IreneGohtami i add useClick here to reset the behavior. wdyt?

Reply:
oh yeah, it's important one, i think it must.